content/articles/easy_to_change.md

```diff
@@ -0,0 +1,32 @@
+title: Easy to Test, Easy to Change
+tags: software engineering
+author: cdent
+created: 2024-02-23
```

Comment:
```suggestion
created: 2024-02-27
```

Thank you! It's [live](https://prepitaph.org/articles/easy-to-change/) now, but I had to change the date (due to my own tardiness here) - I haven't pushed my local changes to give you a chance to do it yourself, if you prefer.

Reply:
I'm happy as long as master ends up being whatever is actually published. So if you're satisfied, coolio.